tests/Alchemy/Tests/Phrasea/Utilities/String/CrLfNormalizerTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Alchemy\Tests\Phrasea\Utilities\String;
+
+use Alchemy\Phrasea\Utilities\StringHelper;
+
+/**
+ * @group functional
+ * @group legacy
+ */
+class CrLfNormalizerTest extends \PhraseanetTestCase
```

Comment:
Can you Merge previous `CamelizerTest` with this one renamed `StringHelperTest` and extending `PHPUnit_Framework_TestCase`?
You could also change group from `functional` to `unit` and remove `legacy` tag. This will speed tests up a little.